src/gate/index.js

```diff
@@ -54,21 +57,46 @@ Object.assign(UserGate.prototype, {
     return this._matchesSample(user) || this._matchesList(user);
   },
 
+  allowsUniform: function(user) {
```

Comment:
If we do collapse this with `allows` we should probably remove the "micro-optimization" comment—I don't actually know which would be faster between hashing the email and checking the list.

Reply:
https://github.com/mixmaxhq/user-gate/pull/5/commits/7e6dfa937363205c20409112b4cd4f9e5553baf0